.github/actions/verify-token/src/predicate.ts

```diff
@@ -69,8 +69,7 @@ export async function createPredicate(
       buildType: DELEGATOR_BUILD_TYPE,
       externalParameters: {
         // Inputs to the TRW, which define the interface of the builder for the
-        // BYOB framework.
-        // TODO(#1575): mask inputs.
+        // BYOB framework. Some of these values may be masked by the caller.
```

Comment:
nit: instead of caller, let's use TRW author - technically caller shouldn't be able to manipulate the provenance, right?

Reply:
+1, I'll use TRW instead. The TRW can manipulate the provenance if they want today, since they provide the inputs to record. (could change if GH context had the information https://github.com/actions/runner/issues/2274#issuecomment-1456475392)

Reply:
done.